README.md

```diff
@@ -1,3 +1,96 @@
+# Node School Vancouver
+
+This is our static site generator, powered by [gatsby](https://www.gatsbyjs.org)
+
+## Attendees
+
+Want a simple pull request? Add yourself to the data/attendees/ directory. Create a new yaml file with your name, with the following content:
+
+```
+---
+name: Name
+github: <github id>
+twitter: <optional twitter id>
+```
+
+When in doubt, look at other people in that directory
+
+## Mentors
+
+If you want to help teach or contribute in anyway, Add yourself to the data/mentors/ directory. Create a new yaml file with your name, with the following content:
+
+```
+---
+name: Name
+github: <github id>
+twitter: <optional twitter id>
+```
+
+We usually like mentors to have come to a session before, and read up on the https://github.com/nodeschool/organizers/wiki/Event-Mentor-Best-Practices#on-being-a-mentor section on the nodeschool wiki.
+
+# Template
+
+This is heavily based off the nodeschool toronto template. It used jekyll which when I tried it wouldn't compile properly, and as a node group, figured all node was better.
```

Comment:
```suggestion
This is heavily based off the NodeSchool Toronto template. It used Jekyll which when I tried it wouldn't compile properly, and as a Node.js group, figured all Node.js was better.
```

Reply:
oh man I tried to remember to capitalize all the nodeschools, but didn't get all of them